docs/Tutorials/Basics/simple-drops.md

```diff
@@ -0,0 +1,225 @@
+3. To install the SDK, simply run the following in your command prompt.  
+```bash
+npm -i keypom-js
+```
+
+---
+
+## Introduction
+:::info
+In this tutorial, you are going to learn how to create a simple drop from scratch. 
+
+A simple drop allows you to onboard users onto NEAR and send them any amount of $NEAR. To learn more on what a simple drop is, [click here](../../Concepts/Keypom%20Protocol/Github%20Readme/Types%20of%20Drops/simpledrops.md)
+:::
+
+Creating a simple drop is a two step process:  
+
+1) Establishing a connection to the NEAR blockchain.  
+2) Creating keypairs and the simple drop.  
+
+The following skeleton code can be used as a starting point:
+``` js
+// Each of the two methods to create this drop will have their own unique set of imports
+
+// Imports used in the Keypom SDK method:
+const { initKeypom, createDrop } = require("keypom-js");
+const { KeyPair, keyStores, connect } = require("near-api-js");
+const path = require("path");
+const homedir = require("os").homedir();
+
+// Imports used in the NEAR-API-JS method:
+const { parseNearAmount, formatNearAmount } = require("near-api-js/lib/utils/format");
+const { KeyPair, keyStores, connect } = require("near-api-js");
+const path = require("path");
+const homedir = require("os").homedir();
+
+
+async function simpleDropKeypom(){
+// STEP 1: Initiate a NEAR connection.
+
+// STEP 2: Create keypairs and the drop.
+}
+
+simpleDropKeypom()
+
+```
+
+---
+
+## Getting Started - Connecting to NEAR
+The first step of creating an Keypom drop is to connect to the NEAR blockchain.
+
+This is done with NEAR-API-JS and consists of creating an unecrypted local keystore and, creating a NEAR configurations, and then using that to initialize a connection to the blockchain. More information about this process can be found [here](https://docs.near.org/tools/near-api-js/quick-reference#key-store).
+
+
+```js reference
+https://github.com/keypom/keypom-js/blob/49244d227f23535ae8962707183d1eca59280d29/docs-examples/keypom-js-sdk/simple-example.js#L8-L25
+```
+
+---
+
+## Creating the Simple Drop
+Now that the connection to the NEAR blockchain is set up, the next step is to create the Keypairs and then create the drop.  
+:::tip
+Recall that the Keypairs that are being generated are the ones that will be embedded in the link given to the user. 
+:::
+
+### The Keypom SDK Approach
+In the SDK, this process starts with calling the `initKeypom` function. This function will always be the first function you need to call to interact with the SDK. 
+
+:::info
+The `initKeypom` function is the first function you need to call in order to interact with the SDK.
+:::
+
+After `initKeypom` is called, the key generation and drop creation can be done. These two tasks are combined into one function, `createDrop`. This function can be either given a set of pre-created keypairs, **or** it can create them automatically.
+
+### The NEAR-API-JS Approach
```

Comment:
Let's scrap all explanations for NAJ and simply have the extra tab that people can reference to see what exactly is happening behind the scenes with the Keypom SDK